src/sentry/api/serializers/models/event.py

```diff
@@ -292,13 +292,19 @@ def __serialize_transaction_attrs(self, attrs, obj):
         """
         Add attributes that are only present on transaction events.
         """
-        return {
+        transaction_attrs = {
             "startTimestamp": obj.data.get("start_timestamp"),
             "endTimestamp": obj.data.get("timestamp"),
             "measurements": obj.data.get("measurements"),
             "breakdowns": obj.data.get("breakdowns"),
         }
 
+        # The _ reflects the temporary nature of this field.
+        if (transaction_metrics_summary := obj.data.get("_metrics_summary")) is not None:
+            transaction_attrs["_metricsSummary"] = transaction_metrics_summary
```

Comment:
i'm okay with this but fwiw i am pretty sure we return this as `_metrics_summary` on the span today. and not as `_metricsSummary`. One extra divergence for UI to deal with.

Reply:
I am aware of that, I did this on purpose since we camelcase all top level fields. But if you want I can use `_metrics_summary`.

Reply:
Had a chat with @ArthurKnaus he suggested to keep the `_metrics_summary`, I have updated the PR.

Reply:
Well, it auto merged, will do in a follow up.